src/components/BookItems.js

```diff
@@ -0,0 +1,8 @@
+const BookItems = () => (
+  <li>
+    <p>No entries yet</p>
```

Comment:
![Screenshot (134)](https://user-images.githubusercontent.com/67087664/149813247-8f204e47-662f-426e-b142-686788381f80.png)

- [ ] I think for now it would be better if you use `props` to render some hardcoded book title and author from `BookList `component.